packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -37,7 +37,7 @@
 }
 
 // The base width of blocks
```

Comment:
Possibly out-of-scope of this PR, but this comment doesn't really make sense. It probably did at some point before a refactor, but now it's just confusing. I think it would be a good idea to update this comment to be more helpful.